at-luantran/Week4/Human_Resources/models/team.rb

```diff
@@ -0,0 +1,26 @@
+require 'mysql2'
+
+class Team
+  attr_accessor :id, :name, :team_lead, :total_members
+  def initialize
+    @connect = Mysql2::Client.new(host: 'localhost', username: 'root',
+                                  password: 'hoichuphong513',
+                                  database: 'LuanTranHumanResources')
+  end
+
+  def create_team(name, team_lead)
+    sql = "INSERT INTO `LuanTranHumanResources`.`team`(name, team_lead, total_members)
+                        VALUES('#{name}', '#{team_lead}', 1)"
+    @connect.query(sql)
+  end
+
+  def get_id_max
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/345833e2b2e3948ab7c1e6ead0c8202d91776721/at-luantran/Week4/Human_Resources/models/team.rb#L17"></span>Naming/AccessorMethodName: Do not prefix reader method names with <code>get_</code>. (https://github.com/rubocop-hq/ruby-style-guide#accessor_mutator_method_names)</td>
    </tr>
  </tbody>
</table>